docs/eventing/broker/kafka-broker.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Apache Kafka Broker"
+weight: 30
+type: "docs"
+---
+
+This page shows how to install and configure the 
+[Kafka Broker](https://github.com/knative-sandbox/eventing-kafka-broker).
+
+This guide assumes a 
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component), only
+`eventing-core` is required.
+
+## Installation
+
+```bash
+kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-broker.yaml" >}}
+```
+
+Verify that `kafka-broker-controller`, `kafka-broker-receiver` and `kafka-broker-dispatcher` are running:
+
+```bash
+kubectl get deployments.apps -n knative-eventing 
+
+NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+eventing-controller            1/1     1            1           10s
+eventing-webhook               1/1     1            1           9s
+kafka-broker-controller        1/1     1            1           3s
+kafka-broker-dispatcher        1/1     1            1           4s
+kafka-broker-receiver          1/1     1            1           5s
+```
+
+## Create a Kafka Broker
+
+A Kafka Broker object looks like this:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    # case-sensitive
+    eventing.knative.dev/broker.class: Kafka
+  name: default
+  namespace: default
+spec:
+  # Configuration specific to this broker.
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: kafka-broker-config
+    namespace: knative-eventing
+  # Optional dead letter sink, you can specify either:
+  #  - deadLetterSink.ref, which is a reference to a Callable
+  #  - deadLetterSink.uri, which is an absolute URI to a Callable (It can potentially be out of the Kubernetes cluster)
+  delivery: 
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: dlq-service
+```
```

Comment:
Move this to the beginning of the doc instead and use it to explain what's required and why in a kafka broker

Reply:
Before the installation?